kubernetes-kit-starter/src/main/java/com/vaadin/kubernetes/starter/KubernetesKitConfiguration.java

```diff
@@ -303,6 +303,9 @@ public HazelcastInstance hazelcastInstance() {
             final var config = new Config();
 
             configure(config);
+            // Make sure Hazelcast shutdown hook is disabled so that the instance
+            // will be stopped after SessionSerializer saved the latest pending state
+            config.setProperty("hazelcast.shutdownhook.enabled", "false");
```

Comment:
If we disable the shutdown hook here, when the `HazelcastInstance` will be closed? Should we add `@Bean(destroyMethod = "shutdown")` to let Spring handle it?

Reply:
We don't need to set the destroyMethod attribute. Spring by convention invokes `shutdown` for bean if existing